### ISSUE TITLE
Translate page slug

### ISSUE THREAD

**Issue by AleCss81** [qTranslate-Team/qtranslate-x#442]
Hi, in these days I'm inspecting the possibility to translate full page permalinks. I have tried to use raw code in the "slug" field of Yoast SEO, but it doesn't work.

Is there a way to have different permalinks for every language?

Thank you very much

**Comment by floopyzicer** [comment]
Yes there is q-translate + q-translate slug and yoast seo will create
sitemaps for each language separately with slugs.
